File: EmployeeManager/EmployeeManager.Shared/DTOs/UpdateEmployeeDTO.cs
using EmployeeManager.Shared.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EmployeeManager.Shared.DTOs
{
    public class UpdateEmployeeDTO
    {
        public string UserID { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public string PhoneNum { get; set; }
        public DateTime DOB { get; set; }
        public bool isDeleted { get; set; }
        public AddressModel Addresss { get; set; } = new AddressModel();
        public List<SkillsModel> Skills { get; set; } = new List<SkillsModel>();

    }
}

File: EmployeeManager/employeemanager.client/src/models/updateEmployeeDTO.ts
export interface UpdateEmployeeDTO {

    userID: string,
    firstName: string,
    lastName: string,
    email: string,
    phoneNum: string,
    DOB: Date,
    isDeleted: boolean,
    addresss: AddressDTO,
    skills: SkillsDTO[],
}

interface SkillsDTO {
    userID: string;
    skillID: string;
    name: string;
    yearsExperience: number;
    seniority: string;
}

interface AddressDTO {
    userID: string;
    streetAddress: string;
    city: string;
    postalCode: string;
    country: string;
}